src/d3a/models/strategy/external_strategies/load.py

```diff
@@ -87,6 +87,15 @@ def _bid(self, payload):
             arguments = json.loads(payload["data"])
             assert set(arguments.keys()) == {'price', 'energy'}
             arguments['buyer_origin'] = self.device.name
+            posted_energy = (arguments['price'] + self.posted_bid_energy(self.market.id)) * 1000.0
```

Comment:
I think you meant `arguments['energy']` here, right?

Reply:
Should be fixed now, thanks!